<?xml version="1.0"?>
<launch>

 <!-- small description about your node -->
 
 <!--<node pkg="my_package_name" type="my_node_name" name="my_node_name"
 respawn="false" output="screen" args="$(find a50_dvl)/config/my_arg_file.yaml"/>-->
 
 <node pkg="a50_dvl" type="a50_dvl_node" name="A50DvlNode" respawn="false" output="screen">
	<rosparam command="load" file="$(find a50_dvl)/config/config_a50_dvl.yaml"/>
</node>

</launch>
